docs/overview/office-add-in-availability.md

```diff
@@ -161,14 +154,7 @@ To work as expected, your Office Add-in might depend on a specific Office host,
         - Content<br>
         - <a href="https://docs.microsoft.com/office/dev/add-ins/reference/requirement-sets/add-in-commands-requirement-sets">Add-in Commands</a></td>
```

Comment:
Need to delete this line (155) also. Add-in Commands are not supported in Office 2016 for Mac (except for Outlook).